src/FeatureFlagsOverriderManager.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Worksome\FeatureFlags;
+
+use Illuminate\Support\Manager;
+use InvalidArgumentException;
+use Worksome\FeatureFlags\Overriders\ConfigOverrider;
+use Worksome\FeatureFlags\Overriders\InMemoryOverrider;
+
+class FeatureFlagsOverriderManager extends Manager
+{
+    public function createConfigDriver(): ConfigOverrider
+    {
+        return new ConfigOverrider(
+            $this->config,
+        );
+    }
+
+    public function createInMemoryDriver(): InMemoryOverrider
+    {
+        /** @var bool|null $overrideAll */
+        $overrideAll = $this->config->get('feature-flags.overriders.in-memory.override-all');
```

Comment:
I think it's safe to remove this doc comment, as we already check below for if it's not `bool` or `null`. 👍🏻
```suggestion
        $overrideAll = $this->config->get('feature-flags.overriders.in-memory.override-all');
```